docs/api-conventions.md

```diff
@@ -55,15 +80,15 @@ Labels are intended for organizational purposes by end users (select the pods th
 
 #### Spec and Status
 
-By convention, the Kubernetes API makes a distinction between the specification of the desired state of a resource (a nested object field called "spec") and the status of the resource at the current time (a nested object field called "status"). The specification is persisted in stable storage with the API object and reflects user input. The status is generated at runtime and summarizes the current effect that the spec has on the system. 
+By convention, the Kubernetes API makes a distinction between the specification of the desired state of an object (a nested object field called "spec") and the status of the object at the current time (a nested object field called "status"). The specification is persisted in stable storage with the API object and reflects user input. The status is summarizes the current state of the object in the system, and may be generated at runtime or persisted by automated processes.
```

Comment:
I'm not opposed to the "generated at runtime" text, but we are moving more towards persisted, with #2726.